modules/EnsEMBL/Draw/GlyphSet/genetree.pm

```diff
@@ -188,23 +189,26 @@ sub _init {
       
     }
 
-    if ($f->{label}) {
-      if( $f->{_genes}->{$other_gene} ){
-        $bold = 1;
-        $label_colour = "ff6666";
-      } elsif( $f->{_genome_dbs}->{$other_genome_db_id} ){
-        $bold = 1;
-      } elsif( $f->{_genes}->{$current_gene} ){
-        $label_colour     = 'red';
-        $collapsed_colour = 'red';
-        $node_colour = 'navyblue';
-        $bold = defined($other_genome_db_id);
-      } elsif( $f->{_genome_dbs}->{$current_genome_db_id} ){
-        $label_colour     = 'blue';
-        $collapsed_colour = 'navyblue';
-        $bold = defined($other_genome_db_id);
+    if ( $f->{label} !~ m/homologs/ ) {
```

Comment:
Is it not possible to merge both if like this: if($f->{label} && $f->{label} !~ m/homologs/)

Reply:
I have tried that before with Perl, but it does not work unlike in JavaScript for some reason. That is why I separated the two conditions. I'll try to merge them both and see if it works with this code though.